### ISSUE TITLE
Labial abscess

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6860]
**Preferred term label:**
Labial abscess

**Synonyms**
 Abscess of the labia

**Definition (free text, please give PubMed ID)**
A circumscribed area of pus or necrotic debris in the labia.
PMID: 7598776

**Parent term (use hpo.jax.org/app)**
Vulvar abscess

**Comment by pnrobinson** [comment]
added
